Api/User.py

```diff
@@ -46,7 +47,7 @@ def post(self):
                             "username":new_user.username,
                             "password":hashed_password
                           })
-        import pdb;pdb.set_trace()
+        # import pdb;pdb.set_trace()
```

Comment:
Leaving debuggers in your code makes your app vulnerable to attack as attackers can see all your logs. Something should be done about this?